docs/getting-started-wsl.md

```diff
@@ -0,0 +1,48 @@
+# Software Setup on Windows Subsystem for Linux
+
+## Table of Contents
+
+* [Table Of Contents](#table-of-contents)
+* [Introduction](#introduction)
+* [WSL2 Setup](#wsl2-setup)
+* [X Server Setup](#wsl2-setup)
+
+
+## Introduction
+
+Windows has a new Windows Subsystem for Linux component that can be used to develop and run code for Linux on Windows. WSL1 was a Windows component that implemented Linux kernel interfaces, and didn't work great with Thunderbots code. WSL2 runs a full-fledged Linux kernel in a VM, and works great with Thunderbots and grSim with the exception that we need to use software rendering instead of GPU-accelerated rendering for grSim.
```

Comment:
Nit:

I might leave out the 'new' because this document may exist for a long time.